python/graphstorm/gconstruct/id_map.py

```diff
@@ -184,26 +186,28 @@ def map_node_ids(src_ids, dst_ids, edge_type, node_id_map, skip_nonexist_edges):
     # If some of the source nodes don't exist in the node set.
     if len(orig_locs) != len(src_ids):
         bool_mask = np.ones(len(src_ids), dtype=bool)
-        bool_mask[orig_locs] = False
+        if len(orig_locs) > 0:
```

Comment:
Can you add some doc string about how map_node_ids handle conner cases like none of src_ids exists in the map.